packages/babel-preset/README.md

```diff
@@ -95,15 +95,14 @@ This packages comes with several different presets for you to use, depending on
 - `@shopify/babel-preset/react`: Adds plugins that transform React (including JSX). You can use this preset with the `@shopify/babel-preset/web` or `@shopify/babel-preset/node` configuration.
 
   This preset accepts an options object.
-  - `hot` : Will automatically add plugins to enable hot reloading of React components. Note that this requires you to have a recent version of `react-hot-loader` installed as a dependency in your project.
   - `pragma` : Replace the function used when compiling JSX expressions. Defaults to `React.createElement`.
   - `pragmaFrag`: Replace the function used when compiling JSX fragment expressions. Defaults to `React.Fragment`.
 
   ```json
   {
     "babel": {
       "presets": [
-        ["@shopify/babel-preset/react", {"hot": true}]
+        ["@shopify/babel-preset/react", {"pragma": "React.createElement"}]
```

Comment:
Why the change to the pragma?

Reply:
Was leaving this as an example of changing the options to the preset. Since hot is no longer an option

Reply:
Showcasing "here's how you change the options from the defaults" by specifying the default value for the option  feels a bit odd to me.

If you feel strongly about having an example here perhaps `{"pragma": "h"}` for preact usage?